helios-sample/src/main/kotlin/helios/sample/sample.kt

```diff
@@ -0,0 +1,50 @@
+package helios.sample
+
+import arrow.core.Either
+import arrow.optics.modify
+import helios.core.Json
+import helios.optics.JsonPath.Companion.root
+import helios.optics.extract
+import helios.optics.selectExtract
+import helios.typeclasses.Decoder
+import helios.typeclasses.DecodingError
+import helios.typeclasses.decoder
+
+const val companyJsonString = """
+{
+  "name": "Arrow",
+  "address": {
+    "city": "Functional Town",
+    "street": {
+      "number": 1337,
+      "name": "Functional street"
+    }
+  },
+  "employees": [
+    {
+      "name": "John doe"
+    },
+    {
+      "name": "Jane doe"
+    }
+  ]
+}"""
+
+fun main(args: Array<String>) {
```

Comment:
This sample is BEGGING to be put into a doc file that's processed by Ank so people can see the results at first glance.